lib/inline_svg/action_view/helpers.rb

```diff
@@ -12,7 +12,7 @@ def inline_svg(filename, transform_params={})
             configured_asset_file.named filename
           end
         rescue InlineSvg::AssetFile::FileNotFound
-          return "<svg><!-- SVG file not found: '#{filename}' #{extension_hint(filename)}--></svg>".html_safe
+          return "<svg class='#{InlineSvg.configuration.svg_not_found_css_class}'><!-- SVG file not found: '#{filename}' #{extension_hint(filename)}--></svg>".html_safe
```

Comment:
Unless you were planning on releasing a new minor version soon, I agree. In a patch it would be far less risky to only add the class if the configuration has been set.

I'll resubmit soon